API.md

```diff
@@ -2716,12 +2716,14 @@ Each request object includes the following properties:
   definition.
 - `url` - the parsed request URI.
 
-#### `request.setUrl(url)`
+#### `request.setUrl(url, [stripTrailingSlash], [qsParserOptions])`
 
 _Available only in `'onRequest'` extension methods._
 
 Changes the request URI before the router begins processing the request where:
  - `url` - the new request path value.
+ - `stripTrailingSlash` - if truthy, strip the trailing slash from the path.
+ - `qsParserOptions` - an object of options to pass to [Qs.parse()](https://www.npmjs.com/package/qs#parsing-objects), which parses the URL's query string, if present.
```

Comment:
Call it something like `parserOptions`. No need to expose the module name in the API.